frontend/src/utils/getUserAgent.ts

```diff
@@ -0,0 +1,4 @@
+export const isMac = (userAgent?: string) => {
```

Comment:
`userAgent`의 기본값으로 `window?.navigator.userAgent`을 넘겨주고,
파라미터를 생략해서 간단하게 사용할 수 있도록 하면 어떨까요?

```suggestion
export const isMac = (userAgent = window?.navigator.userAgent) => {
```

Reply:
오 좋습니다! 반영해두었어요~([d14c8a8](https://github.com/GooJinSun/diivers/pull/277/commits/d14c8a8b22b2c150141033558ed5c2bce88df6ae))